packages/plugin-vue/src/main.ts

```diff
@@ -58,6 +58,7 @@ export async function transformMain(
   // inlined template cannot be individually hot updated.
   const useInlineTemplate =
     !devServer &&
+    !devToolsEnabled &&
```

Comment:
Seems exposing `__filename` is enough for the devtool to display the component name, why do you want to disable `inlineTemplate`?

Reply:
Because for component with script setup, inlineTemplate option doesn't expose any setup binding. Vue devtools relies on this to display component setup binding correctly.

Reply:
Got it.
I have discussed this with Evan today.
As the `inlineTemplate` has an impact on the app performance, we believe that it shouldn't be disabled.
We need to find another way to fix this issue. Maybe in another package.